### PR TITLE
rospy.sleep exception handling

### PR DESCRIPTION
**Pull request by fmessmer**
some `sleep()` exception handling which prevents a lot of terminal output on "STRG+C"

### CODE REVIEW

File: cob_dashboard/scripts/cob_dashboard_aggregator.py [hunk 1]
```diff
@@ -75,4 +75,8 @@ def publish(self):
   r = rospy.Rate(1)
   while not rospy.is_shutdown():
     da.publish()
-    r.sleep()
+    try:
+        r.sleep()
+    except rospy.exceptions.ROSInterruptException as e:
+        #print "ROSInterruptException", e
```
Comment (mgruhler) [inline cob_dashboard/scripts/cob_dashboard_aggregator.py]: why not remove this line (same below)?

Comment (fmessmer) [reply cob_dashboard/scripts/cob_dashboard_aggregator.py]: copied it from some other script which had it...

**Review by floweisshardt — APPROVED** [review]

File: cob_dashboard/scripts/cob_dashboard_aggregator.py (hunk 1 above)
Comment (fmessmer) [reply cob_dashboard/scripts/cob_dashboard_aggregator.py]: convinced: it's cleaner without

**Comment by fmessmer** [comment]
Added proper `from __future__ import print_function` as suggested in https://github.com/ipa320/cob_control/issues/129#issuecomment-274029626 and [python_2_and_3_compatible_code](http://wiki.ros.org/python_2_and_3_compatible_code#Print_is_now_a_function)

**Review by NadiaHG — APPROVED** [review]

**Pull request merged**